src/utils/getSimilarActionInQueue.js

```diff
@@ -12,12 +12,15 @@ export default function getSimilarActionInQueue(
   actionQueue: Array<*>,
 ) {
   if (typeof action === 'object') {
-    return actionQueue.find((queued: *) => isEqual(queued, action));
+    return actionQueue.find((queued: *) => _.isEqual(queued, action));
   }
   if (typeof action === 'function') {
-    return actionQueue.find(
-      (queued: *) => action.toString() === queued.toString(),
-    );
+    return actionQueue.find((queued: *) => {
+      const isArgsEqual = _(action.meta.args)
+        .xorWith(queued.meta.args, _.isEqual)
```

Comment:
Will comparing `args` one by one better? I think `_.xorWith` will treat `[1, 2]` with `[2, 1]` the same. What's more, with #208, users can customize their own `comparisonFn`.